tools/import-image-tests

```diff
@@ -147,15 +157,43 @@ def main():
     if args.db_ignore:
         with open(args.db_ignore, "r", encoding="utf-8") as f:
             db_ignore = f.read().splitlines()
+        print("Filtering manifests, ignoring:")
+        for dbi in db_ignore:
+            print(f"    - {dbi}")
+
+    filters = []
+    if args.filter_with_ci_distros:
+        print("Filtering distros, importing only:")
+        with open(args.filter_with_ci_distros, "r", encoding="utf-8") as f:
+            data = yaml.load(f, Loader=yaml.Loader)
+            runners = data["Manifests"]["parallel"]["matrix"]
+            for runner in runners:
+                for machine_id in runner["RUNNER"]:
+                    flavor = machine_id.split("/")[-1]
+                    arch = flavor.split("-")[-1]
+                    distro = ""
+                    if "fedora" in flavor:
+                        distro = f"fedora-{flavor.split('-')[1]}"
+                    elif "centos" in flavor:
+                        distro = f"centos-{flavor.split('-')[2]}"
+                    elif "rhel" in flavor:
+                        distro = f"rhel-{flavor.split('-')[1].replace('.', '')}"
+                    print(f"    - {distro} {arch}")
+                    filters.append((distro,arch))
```

Comment:
I'd probably do this as a `dict[str, list[str]]` but that's just me running out of things to say ;)